docker-compose.yml

```diff
@@ -34,7 +34,7 @@ services:
       - "5432"
 
   redis:
-    image: redis
+    image: redis:3.0.3
```

Comment:
is this the redis version we're using on prod? I just checked and I have 5.0.7 installed locally, and this version doesn't seem to work properly on my laptop

Reply:
3.0.3 is also [almost five years old](https://github.com/antirez/redis/releases/tag/3.0.3) as far as I can tell, so we may want to shoot for a newer version

Reply:
Yes, I'll use 5.0.7. I think I had used the version that micromasters was pinned to